docs/modules/c-sharp/index.md

```diff
@@ -253,6 +253,17 @@ public partial struct Person
 }
 ```
 
+You can create multiple tables backed by items of the same type by applying it with different names. For example, to store active and archived posts separately and with different privacy rules, you can declare two tables like this:
+
+```csharp
+[SpacetimeDB.Table(Name = "Posts", Public = true)]
+[SpacetimeDB.Table(Name = "ArchivedPosts", Public = false)]
```

Comment:
Table names should be singular, not plural.
```suggestion
[SpacetimeDB.Table(Name = "Post", Public = true)]
[SpacetimeDB.Table(Name = "ArchivedPost", Public = false)]
```

Reply:
Sigh. As a user, I still disagree with it and would choose plural, but fine :P